compiler/env/OMRCPU.cpp

```diff
@@ -29,15 +29,32 @@
 #include "env/defines.h"
 #include "omrcfg.h"
 
+namespace OMR
+{
+
+CPU::CPU():
+   _processor(TR_NullProcessor),
+   _endianness(TR::endian_unknown),
+   _majorArch(TR::arch_unknown),
+   _minorArch(TR::m_arch_none)
+   {
+   OMRPORT_ACCESS_FROM_OMRPORT(TR::Compiler->portLib);
+   omrsysinfo_get_processor_description(&_processorDescription);
+
+   for (int i = 0; i < OMRPORT_SYSINFO_FEATURES_SIZE; i++)
+      {
+      _featureMasks[i] = ~0;
+      }
+   }
+
 TR::CPU *
-OMR::CPU::self()
+CPU::self()
```

Comment:
Curious, it looks like `CPU` is supposed to be an extensible class but is not marked `OMR_EXTENSIBLE`. That could be very dangerous.